### PR TITLE
Fix normalization factor math formulas in multivariate_normal docstrings

### LINKED ISSUE #38615: Potential mismatch regarding documentation strings of JAX and SciPy
URL 1:
https://github.com/jax-ml/jax/blob/0b8afd089fa19111114d9078767713506de88197/jax/_src/scipy/stats/multivariate_normal.py

URL 2:
https://github.com/scipy/scipy/blob/9434822785d08577b2279428cbad2dce7eb8dbb9/scipy/stats/_multivariate.py

textual source 1:
line 86 (or line 36) of a resource at URL 1

textual source 2:
lines 396 and 397 of a resource at URL 2

Textual source 1 does not seem to contain "sqrt", whereas textual source 2 seems to contain "sqrt".

### PR DESCRIPTION
**Pull request by Kinetic-Labs-GT**
### Problem
As noted in #38615, the documentation strings for `jax.scipy.stats.multivariate_normal.pdf` and `jax.scipy.stats.multivariate_normal.logpdf` contained a math notation error. The normalization factor in the LaTeX block omitted the square root term over the denominator, diverging from the standard formulation and upstream SciPy's reference documentation.

### Solution
Updated the `.. math::` blocks in both `pdf` and `logpdf` functions within `jax/_src/scipy/stats/multivariate_normal.py` to wrap the denominator term inside a `\sqrt{}` block:
$$\frac{1}{\sqrt{(2\pi)^k\det\Sigma}}$$

This is a pure documentation fix; the underlying runtime execution paths are mathematically correct and remain unchanged.

### Verification
- Validated that the file compiles cleanly via `py_compile`.
- Ran the local `pre-commit` hook suite to ensure style compliance.
- Ran the relevant unit tests to verify no regressions:
  ```bash
  pytest tests/scipy_stats_test.py
Result: All 972 passed.

Fixes #38615

### CODE REVIEW

**Review by jakevdp — APPROVED** [review]
Thanks!

**Comment by jakevdp** [comment]
Please squash your changes into a single commit – thanks!

**Comment by Kinetic-Labs-GT** [comment]
Hi @jakevdp, 

Done! I have successfully squashed the changes into a single clean commit and force-pushed. The PR is ready for review. 

Thanks again!

**Review by jakevdp — APPROVED** [review]
